.github/workflows/regression-report.yml

```diff
@@ -19,12 +19,7 @@ env:
   # yamllint disable-line rule:line-length
   DEFAULT_PROJECTS_LINK: "https://raw.githubusercontent.com/checkstyle/test-configs/main/extractor/src/main/resources/list-of-projects.properties"
   EXTRACTOR_VERSION: 2024-08-27
-  CACHE_KEY_PR: "simple-cache-key-PR-${{ github.sha }}"
-  CACHE_KEY_MASTER: "simple-cache-key-MASTER-${{ github.sha }}"
-  CACHE_KEY_CONTRIBUTION: "simple-cache-key-CONTRIBUTION-${{ github.sha }}"
-  CACHE_KEY_CONFIG_FILES: "config-files-${{ github.sha }}-${{ github.event.comment.id }}"
-  # yamllint disable-line rule:line-length
-  CACHE_KEY_GENERATED_CONFIGS: "generated-configs-${{ github.sha }}-${{ github.event.comment.id }}"
+  CACHE_KEY: "checkouts-${{ github.sha }}-${{ github.event.comment.id }}"
```

Comment:
`regression-report-....`

to make it clear who owns this cache

Reply:
can you elaborate this, is it to change the name of the cache-key?